src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -3228,6 +3135,101 @@ paths:
           description: request specified an invalid format
         '500':
           description: internal server error
+
+  /timestamping-services:
```

Comment:
The indentation seems to be 4 spaces instead of 2 in this part. The openapi doc should probably be indented consistently with 2 spaces. 

I have noticed that editing yaml with IntelliJ IDEA makes it really really difficult to indent with 2 spaces so I have used other editors such as Sublime Text 3 or VSCode for that.

Reply:
You're right, the indentation was incorrect. That's fixed now.